packages/components/src/Banner/Banner.tsx

```diff
@@ -9,7 +9,7 @@ import NotificationIcon, {
 } from "../common/Notifications/NotificationIcon";
 import styles from "./Banner.module.css";
 
-type Props = {
+type BannerProps = {
```

Comment:
```suggestion
export type BannerProps = {
```

we should also add to `Banner/index.ts`:
```
export type { BannerProps } from "./Banner";
```

that way we can import from `lib/Banner` instead of needing `lib/Banner/Banner`

Reply:
One way you could test is making sure within `index.ts` you can do
```
import {BannerProps} from "./Banner";
```